cmd/eksctl-anywhere/cmd/copypackages.go

```diff
@@ -0,0 +1,120 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/manifests/bundles"
+	"github.com/aws/eks-anywhere/pkg/registry"
+	releasev1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+// copyPackagesCmd is the context for the copy packages command.
+var copyPackagesCmd = &cobra.Command{
+	Use:          "packages",
+	Short:        "Copy package images and charts from a source to a destination",
+	Long:         `Copy all the EKS Anywhere package images and helm charts from a source to a destination.`,
+	SilenceUsage: true,
+	RunE:         runCopyPackages,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if err := cobra.ExactArgs(1)(cmd, args); err == nil {
+			return nil
+		}
+		return fmt.Errorf("A destination must be specified as an argument")
+	},
+}
+
+func init() {
+	copyCmd.AddCommand(copyPackagesCmd)
+
+	copyPackagesCmd.Flags().StringVarP(&copyPackagesCommand.bundleFile, "bundle", "b", "", "EKS-A bundle file to read artifact dependencies from")
+	if err := copyPackagesCmd.MarkFlagRequired("bundle"); err != nil {
+		log.Fatalf("Cannot mark 'bundle' flag as required: %s", err)
+	}
+	copyPackagesCmd.Flags().StringVarP(&copyPackagesCommand.dstCert, "dst-cert", "", "", "Package bundles file to read artifact dependencies from")
```

Comment:
Looks like some sort of cut and paste errors? Both here on line 39 and on line 40?